src/main/java/Duke.java

```diff
@@ -1,10 +1,178 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+
     public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line = "________________________________________";
+        String curCommand = "";
+
+        readFromFile();
+        writeIntoFile();
+
+        System.out.println(logo);
+        System.out.println(line);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        System.out.println(line);
+
+        while (!curCommand.equals("bye")) {
+            curCommand = input.nextLine();
+            System.out.println(line);
+            handleCommand(curCommand);
+            System.out.println(line);
+        }
     }
+
+    // Handle command
```

Comment:
Such two-word comments arent particularly useful. Try to make your comments more descriptive